src/core/Akka/Serialization/Serialization.cs

```diff
@@ -119,6 +119,26 @@ public Serialization(ExtendedActorSystem system)
             }
         }
 
+        private Serializer GetSerializerByName(string name)
+        {
+            if (name == null)
+                return null;
+
+            var serializersConfig = System.Settings.Config.GetConfig("akka.actor.serializers").AsEnumerable().ToList();
+            foreach (var kvp in serializersConfig)
```

Comment:
Because this config is not saved in the Serialization subsystem, I have to retrieve this value again. Leaning on the assumption of hocon config caching here.